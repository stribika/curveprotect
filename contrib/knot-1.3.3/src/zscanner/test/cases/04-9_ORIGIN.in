$TTL	1

; KO
$ORIGIN 123456789012345678901234567890123456789012345678901234567890123.123456789012345678901234567890123456789012345678901234567890123.123456789012345678901234567890123456789012345678901234567890123.12345678901234567890123456789012345678901234567890123456789012.	; Domain name exceeded maximal length
