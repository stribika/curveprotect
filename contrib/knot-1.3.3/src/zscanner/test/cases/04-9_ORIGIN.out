ERROR=ZSCANNER_EDNAME_OVERFLOW
------
